docs/installation.rst

```diff
@@ -115,7 +115,7 @@ to install the plugins as well.
 To build the client-side code project, cd into the root of the repository
 and run: ::
 
-    npm install
+    npm install && npm run build
 
 This will run multiple `Grunt <http://gruntjs.com>`_ tasks, to build all of
```

Comment:
We're still using Grunt, right?  Either way, we should make sure to keep a close look at the docs as we make bigger changes.